### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: nginx/conf.d/app.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+server {
+  listen 80;
+  index index.js;
```
Comment (rudigiesler) [inline nginx/conf.d/app.conf:3]: Is the index not `index.html`?

File: nginx/conf.d/app.conf [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  server_name _;
+  root /usr/share/nginx/html;
```
Comment (rudigiesler) [inline nginx/conf.d/app.conf:5]: Looking at the docs at https://hub.docker.com/_/nginx/ , I think we should just be able to do a `COPY --from=node app/out /usr/share/nginx/html`, and not require any configuration?

Comment (KaitCrawford) [reply nginx/conf.d/app.conf:5]: ah ok. so you don't think we'll need this file?

Comment (rudigiesler) [reply nginx/conf.d/app.conf:5]: Yeah, I think we can make it much simpler and remove this file and the dockerfile line that copies this folder.

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**